src/uct/sm/mm/sysv/mm_sysv.c

```diff
@@ -65,8 +65,18 @@ uct_sysv_mem_alloc(uct_md_h tl_md, size_t *length_p, void **address_p,
     uct_mm_md_t *md = ucs_derived_of(tl_md, uct_mm_md_t);
     ucs_status_t status;
     uct_mm_seg_t *seg;
+    uct_md_attr_t md_attr;
     int shmid;
 
+    status = uct_sysv_md_query(tl_md, &md_attr);
+    if (status != UCS_OK) {
+        return status;
+    }
+
+    if (!(md_attr.cap.alloc_mem_types & UCS_BIT(mem_type))) {
+        return UCS_ERR_UNSUPPORTED;
+    }
+
```

Comment:
no need to query. sysv supports only HOST

Reply:
done